__tests__/html/transcript.legacyActivityMiddleware.reactionButtons.html

```diff
@@ -85,26 +81,21 @@
         return next(...args);
       };
 
-      (async function() {
+      run(async function () {
```

Comment:
Verification: reaction buttons are not rendering; is that ok?

Reply:
You mean the tofu, right?

Will update it to U/D.

![image](https://user-images.githubusercontent.com/1622400/117732340-8d66a780-b1a4-11eb-80ec-b5716d562df5.png)

Reply:
Updated to:

![image](https://user-images.githubusercontent.com/1622400/117732554-f5b58900-b1a4-11eb-8f1a-0cff05e65bf1.png)